### ISSUE TITLE
[Legitimate Site Blocked] www.metapass.cloud

### ISSUE THREAD

**Issue by KEW345** [MetaMask/eth-phishing-detect#6063]
https://www.metapass.cloud/
hey what's up I am just building this site.
I would like it to have a metamask password if you could help with that.
I am not pushing to any site .
just links to help every one get to sites.

Please unblock my site from you meta mask block.

**Comment by devenblake** [comment]
Fixed by #6074.

**Comment by devenblake** [comment]
Closing because it looks like this has been fixed, feel free to re-open or file a new issue if the problem persists.
